docs/community/communication.mdx

```diff
@@ -0,0 +1,95 @@
+  - **Working and interest group discussions** (`#client-implementors`, `#agents-wg`, etc.)
+    - **Working Group**: Some specific goal or project in mind (such as an SDK, inspector, registry, server-identity, load-balancing, etc).
+    - **Interest Group**: An abstract gathering of folks that might raise a range of various topics. Some might get actioned on as one-offs, others might spin into Working Groups.
+  - **Community onboarding** and contribution guidance.
+  - **Community feedback** and collaborative brainstorming.
+  - Public **office hours** and **maintainer availability**.
+- Unapproved uses:
+  - MCP user support: participants are expected to to read official documentation and start new GitHub Discussions for questions or support.
+  - Service or product marketing: interactions on this Discord are expected to be vendor-neutral and not used for brand-building or sales. Mentions of brands or products are discouraged outside of being used as examples or responses to conversations that start off focused on the specification.
+
+#### Private channels (Exceptions)
+
+- **Purpose**: Confidential coordination and sensitive matters that cannot be discussed publicly. Access will be restricted to designated maintainers.
+- **Strict criteria for private use**:
+  - **Security incidents** (CVEs, protocol vulnerabilities).
+  - **People matters** (maintainer-related discussions, code of conduct policies).
+  - Select channels will be configured to be **read-only**. This can be good for example for maintainer decision making.
+  - Coordination requiring **immediate** or otherwise **focused response** with a limited audience
+- **Transparency requirements**:
+  - **All decisions** affecting the community **must be documented** in GitHub Discussions and/or Issues, and will be labeled with `notes`.
+  - Private channels are to be used as **temporary "incident rooms,"** not for routine development.
+
+Any significant discussion on Discord that leads to a potential decision or proposal must be moved to a GitHub Discussion or GitHub Issue to create a persistent, searchable record. Proposals will then be promoted to full-fledged PRs with associated work items (GitHub Issues) as needed.
+
+### GitHub Discussions
+
+For structured, long-form updates on project direction, features, and community topics.
+
+When to use:
+
+- Formal governance decisions requiring community input and voting
+- Project roadmap planning and milestone discussions
+- Announcements and release communications
+- Community polls and consensus-building processes
+
+### GitHub Issues
+
+For bug reports, feature tracking, and actionable development tasks.
+
+When to use:
+
+- Submit SEP proposals (following the [SEP guidelines](./sep-guidelines))
+- Bug reports with reproducible steps
+- Feature requests with context and rationale
```

Comment:
Pursuant to https://github.com/modelcontextprotocol/modelcontextprotocol/pull/1002#discussion_r2228794078.

```suggestion
```